src/app/components/DynamicProgress.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
-import { Progress, Icon } from 'react-components';
+import { Icon } from 'react-components';
 import contactLoader from 'design-system/assets/img/pm-images/contact-export-loader.gif';
 
-const DyanmicProgress = ({ id, alt, displayDuring, displayEnd, value, max, ...rest }) => {
+const DyanmicProgress = ({ id, alt, displayDuring, displayEnd, value, max, endPostponed, ...rest }) => {
```

Comment:
```suggestion
const DynamicProgress = ({ id, alt, displayDuring, displayEnd, value, max, endPostponed, ...rest }) => {
```

Reply:
Good catch! Because of the `export default` this went unnoticed...